src/components/pagination/_pagination.scss

```diff
@@ -237,12 +245,30 @@ $css--helpers: true;
 
     .#{$prefix}--select__item-count {
       border-right: $spacing-4xs solid $ui-03;
-      padding-right: 0.5rem;
     }
 
     .#{$prefix}--select__page-number {
       border-left: $spacing-4xs solid $ui-03;
-      padding-left: 0.5rem;
+    }
+
+    .#{$prefix}--select__item-count .#{$prefix}--select-input {
+      @include breakpoint($pagination-breakpoint) {
```

Comment:
`breakpoint` mix-in is a deprecated API and I'm planning to removing it from `v10`, which connects to the point @joshblack had. @joshblack would you be open to support custom breakpoint in your breakpoint API?

Reply:
cool thank you! i updated these based on our convo from yesterday